### PR TITLE
Guil8553/doc warnings

### PR DESCRIPTION
**Pull request by GuillaumeBelz**
Some doc fixes

### CODE REVIEW

File: uitools/examples/qml/demos/AuthenticationViewDemoForm.qml [hunk 1]
```diff
@@ -25,9 +25,11 @@ DemoPage {
         MapView {
             id: view
 
+            //! [Set up Authentication View QML]
```
Comment (GuillaumeBelz) [inline uitools/examples/qml/demos/AuthenticationViewDemoForm.qml:28]: Necessary to have this when `\snippet` is used

File: uitools/toolkitcpp/import/Esri/ArcGISRuntime/Toolkit/AuthenticationView.qml [hunk 2]
```diff
@@ -49,7 +49,7 @@ import QtQuick.Controls
         {Qt WebView}.
 
   \image docs/authenticationview_qml.png authentication_view
-  \snippet cpp_quick/src/demos/AuthenticationViewDemoForm.qml Set up Authentication View QML
+  \snippet qml/demos/AuthenticationViewDemoForm.qml Set up Authentication View QML
```
Comment (GuillaumeBelz) [inline uitools/toolkitcpp/import/Esri/ArcGISRuntime/Toolkit/AuthenticationView.qml:52]: Fix path, which was changed when QML API was removed

File: uitools/toolkitcpp/import/Esri/ArcGISRuntime/Toolkit/Callout.qml [hunk 3]
```diff
@@ -39,17 +39,22 @@ import QtQuick.Shapes
...
      For more information, please see the CalloutData documentation.
      \image docs/callout.png
-     \snippet cpp_quick/src/demos/CalloutDemoForm.qml Set up Callout QML
+     \snippet qml/demos/CalloutDemoForm.qml Set up Callout QML
 
      \note That the Callout has gone through a major revision as of ArcGISRuntime 100.14.
      Part of this revision has been a change to the styling behaviour of the Callout, making the Callout
      compliant with your currently applied theme. To revert to the classic Callout look, you can supply the
      old style properties to the Callout as provided below.
-     \oldcode
```
Comment (GuillaumeBelz) [inline uitools/toolkitcpp/import/Esri/ArcGISRuntime/Toolkit/Callout.qml:48]: `\oldcode` and `\newcode` were removed. Just use `\code`

File: uitools/toolkitcpp/import/Esri/ArcGISRuntime/Toolkit/Callout.qml [hunk 4]
```diff
@@ -39,17 +39,22 @@ import QtQuick.Shapes
...
+
+     \code
             Callout {
               calloutData: myCalloutData
             }
-     \newcode
+     \endcode
+
+     can be rewrite as:
```
Comment (bri12415) [inline uitools/toolkitcpp/import/Esri/ArcGISRuntime/Toolkit/Callout.qml:55]:
```suggestion
     can be rewritten as:
```

**Review by bri12415 — APPROVED** [review]

**Pull request merged**